### PR TITLE
Master

### PR DESCRIPTION
**Pull request by khowe**

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm [hunk 1]
```diff
@@ -387,6 +387,7 @@ sub resource_classes {
          '64Gb_job'     => {'LSF' => '-C0 -M64000 -R"select[mem>64000] rusage[mem=64000]"' },
          '512Gb_job'     => {'LSF' => '-C0 -M512000 -R"select[mem>512000] rusage[mem=512000]"' },
 
+         '8Gb_8c_job'  => {'LSF' => '-n 8 -C0 -M8000  -R"select[mem>8000]  rusage[mem=8000]  span[hosts=1]"' },
```
Comment (mateuspatricio) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm]: Resources classes should be only added on the Example/ pipeline config files.

Comment (khowe) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm]: But the template in ProteinTrees_conf.pm (which is commented out in a POD block) defines which resource classes need to be defined. This one was missing from the template.

File: modules/Bio/EnsEMBL/Compara/PipeConfig/Example/EGProteinTrees_conf.pm [hunk 2]
```diff
@@ -280,6 +281,7 @@ sub resource_classes {
          '64Gb_job'     => {'LSF' => '-q production-rh6 -M64000 -R"select[mem>64000] rusage[mem=64000]"' },
          '512Gb_job'     => {'LSF' => '-q production-rh6 -M512000 -R"select[mem>512000] rusage[mem=512000]"' },
 
+         '8Gb_8c_job' => {'LSF' => '-q production-rh6 -n 8 -C0 -M8000 -R"select[mem>8000] rusage[mem=8000] span[hosts=1]"' },
```
Comment (mateuspatricio) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Example/EGProteinTrees_conf.pm]: Is this resource being used? Perhaps it was in a different commit?

Comment (khowe) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Example/EGProteinTrees_conf.pm]: Yes - in the prottest_8_cores analysis

Comment (khowe) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Example/EGProteinTrees_conf.pm]: Hi @mateuspatricio any news on this? @uma-ensembl was in the process of creating a pull request for the same thing (although she intended to do it on release/87)

Comment (mateuspatricio) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/Example/EGProteinTrees_conf.pm]: Hi @khowe, sorry I didn't see your reply.
Anyways it was fine and @muffato has approved the pull request.

**Pull request merged**